pyproject.toml

```diff
@@ -31,15 +31,16 @@ known-first-party = ["django_stubs_ext", "mypy_django_plugin"]
 split-on-trailing-comma = false
 
 [build-system]
-requires = ["setuptools"]
-build-backend = "setuptools.build_meta"
+requires = ["hatchling"]
```

Comment:
No idea what `hatchling` is. 

If other maintainers also don't know what this is, I prefer not to rely on it :(

Reply:
I'll test again with setuptools then :)

Reply:
I've used `hatchling` for other projects, a totally fair build system that does its job. It also comes with a sort of ecosystem for package management called `hatch`, which is completely __optional__ and I've _only_ used the build system named `hatchling`.

One thing I did like about it was that it validates the `project` section in your `pyproject.toml`, which was quite nice.  I found that the documentation for `hatchling` was quite simple to pick up on too, here: https://hatch.pypa.io/latest/config/build/

It's maintained by https://github.com/pypa/ and e.g. https://github.com/pypa/virtualenv/ is packaged with `hatchling`: https://github.com/pypa/virtualenv/blob/9eac8a6b85c047e2287c95bdb9caf6f0fbf35041/pyproject.toml#L1-L6

It's nice to extend `hatchling` with `hatchling-vcs` to get versioning via version control(`git`). Should be similar to `setuptools-vcs`. Which is 1 less thing to remember/do when crafting a release.

---

I'm +1 on using `hatchling`

Reply:
Here are their own reasons why to use `hatchling` as build backend instead of setuptools:

https://hatch.pypa.io/latest/why/#build-backend